GeneralsMD/Code/Libraries/Source/WWVegas/WW3D2/texturethumbnail.cpp

```diff
@@ -261,6 +260,7 @@ ThumbnailClass::~ThumbnailClass()
 
 void ThumbnailManagerClass::Create_Thumbnails()
 {
+#if 0
```

Comment:
Why is this disabled?

Reply:
depended on mixfile, the game doesn't use mixes, for sake of having a clean pr i just did this as rest of thumbnail code is called